packages/ntopng/PKGBUILD

```diff
@@ -0,0 +1,33 @@
+# This file is part of BlackArch Linux ( https://www.blackarch.org/ ).
+# See COPYING for license details.
+
+pkgname=ntopng
+pkgver=3.8
+pkgrel=1
+pkgdesc='Web-based Traffic and Security Network Traffic Monitoring.'
+arch=('any')
+groups=('blackarch')
+url='https://www.ntop.org/'
+license=('GPL-3.0')
+depends=('gcc' 'libmaxminddb' 'nDPI')
+makedepends=('nDPI')
+source=("https://github.com/ntop/ntopng/archive/3.8-stable.zip")
+sha512sums=('SKIP')
+
+prepare() {
+   cd "$pkgname-$pkgver-stable"
+   ./autogen.sh
+}
+
+build() {
+  ./install_nDPI.sh
```

Comment:
Why are you doing that? You can't just run any `sh` script in the PKGBUILD, as I said, if you need ndpi for the package, then you need to package it using lowercase names.